samples/samples_for_tutorials/product/add_fulfillment_places.js

```diff
@@ -18,7 +18,7 @@ async function main(generatedProductId) {
   // [START retail_add_remove_fulfillment_places]
 
   // Imports the Google Cloud client library.
-  const { ProductServiceClient } = require('@google-cloud/retail').v2;
+  const {ProductServiceClient} = require('@google-cloud/retail').v2;
   const utils = require('../setup/setup_cleanup');
 
   const projectNumber = process.env['PROJECT_NUMBER'];
```

Comment:
'apiEndpoint' is assigned a value but never used. Maybe for other files we can remove apiEndpoint as well since this is the default value.

Reply:
done